apps/public-docsite-v9/.storybook/main.js

```diff
@@ -15,6 +15,7 @@ module.exports = /** @type {Omit<import('../../../.storybook/main'), 'typescript
     '../../../packages/react-components/react-migration-v0-v9/stories/**/@(index.stories.@(ts|tsx)|*.stories.mdx)',
     '../../../packages/react-components/react-migration-v8-v9/stories/**/@(index.stories.@(ts|tsx)|*.stories.mdx)',
     '../../../packages/react-components/react-datepicker-compat/stories/**/@(index.stories.@(ts|tsx)|*.stories.mdx)',
+    '../../recipes-react-components/src/recipes/**/*.stories.mdx',
```

Comment:
hmm this feels wrong from domain boundary POV. an app should never use things from another app. In future we will have in place lint rules that would not allow this. 

Don't wanna block the progress here thus can you please create issue to fix this going forward ?

Reply:
In that case, what would be the way to go with this? should this be a package instead to avoid this issue?

Reply:
yeah, AFAIR the intent of recipes was to be as standalone app correct ? but if we wanna make it part of v9 docs it is no longer true. 

We still can have it as an app to be published standalone though, if there is such a need etc.

NOTE: make sure to make the library private as we don't wanna expose it to npm registry.

Reply:
Hmm I'll talk with everyone else, but as I understand we wanted it to be pretty visible in the docs since we will use them as placeholder recipes for components that we might work on later. I'll change it to draft for now.

Reply:
nothing will change in terms of "visibility"

it will only change import.

` '../../../packages/react-components/src/recipes/**/*.stories.mdx',`

then the app will can still exist but with dependency on library

`recipes-app` --> `recipes-lib`

Reply:
so there's no problem having the package never released and private? I think that was a concern that made us go with the app instead of package. This package wouldn't follow semver as well since we'll allow breaking changes and such. If those things aren't problems, then I'll create a PR to move it to a package in the coming days.

Reply:
yeah I dont see any problem with that. if it's  gonna be private even better. ty!